src/backend.rs

```diff
@@ -131,6 +131,7 @@ pub(crate) enum AppStateUpdate<'s> {
     ),
     IdentityRegistrationProgressed, // TODO provide state update details
     LoadedIdentity(MappedMutexGuard<'s, Identity>),
+    LoadedKnownIdentity(MappedMutexGuard<'s, Identity>),
```

Comment:
Is it any different from `LoadedIdentity`? Like do we care which way identity came